open-ended-capstone/second_etl.py

```diff
@@ -0,0 +1,194 @@
+import random
```

Comment:
there are lines of code that do nothing

like.  `create_table_trg(trg_conn, None)`